### PR TITLE
Jcn local

### PR DESCRIPTION
**Pull request by jeancheikh**
firstName and lastName fields replaced by name.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -24,4 +24,7 @@ app.use("/", indexRouter);
 app.use("/users", usersRouter);
 app.use("/courses", coursesRouter);
 
+const port = 7000;
```
Comment (mzerbo98) [inline app.js]: Please, could you remove these parts with listening ? It could causes some problems and it's not necessary

**Review by mzerbo98 — CHANGES_REQUESTED** [review]
Could you also correct codacy errors ?

File: app.js [hunk 2]
```diff
@@ -24,4 +24,6 @@ app.use("/", indexRouter);
 app.use("/users", usersRouter);
 app.use("/courses", coursesRouter);
 
+let server = app.listen(3000, () => console.log('Listening on port: ', server.address().port))
```
Comment (mzerbo98) [inline app.js]: In fact, there's another file which already start the app, it's bin/www. There's no need to have a app.listen here

Comment (mzerbo98) [reply app.js]: That file is launched with npm start

**Review by mzerbo98 — CHANGES_REQUESTED** [review]

File: app.js (hunk 2 above)
Comment (jeancheikh) [reply app.js]: Thanks

**Review by mzerbo98 — DISMISSED** [review]
There still errors with Codacy but except that, it looks good to me

**Review by Lenscorpx — DISMISSED** [review]

**Review by atoo — CHANGES_REQUESTED** [review]
Please fix all Codacy issues https://app.codacy.com/gh/BuildForSDGCohort2/Team-004-Backend/pullRequest?prid=6207256

**Review by mzerbo98 — DISMISSED** [review]
Looks Good to me

**Review by Lenscorpx — APPROVED** [review]

**Review by Lenscorpx — DISMISSED** [review]

**Review by atoo — DISMISSED** [review]
Approved based on the 2 other approvals.

**Comment by atoo** [comment]
@jeancheikh you have 1 issue to fix https://app.codacy.com/gh/BuildForSDGCohort2/Team-004-Backend/pullRequest?prid=6207256, and i'm still getting 404.

Can you take the necessary time to carefully test you work?

**Review by mzerbo98 — DISMISSED** [review]

**Review by mzerbo98 — DISMISSED** [review]
Looks good to me except the codacy errors

**Comment by atoo** [comment]
Yes please @jeancheikh fix the issues and it is done https://app.codacy.com/gh/BuildForSDGCohort2/Team-004-Backend/pullRequest?prid=6207256

**Review by mzerbo98 — APPROVED** [review]
LGTM

**Review by Lenscorpx — APPROVED** [review]
LGTM

**Review by atoo — APPROVED** [review]
Approved based on the other approvals.

**Comment by atoo** [comment]
Thank you

**Pull request merged**